### PR TITLE
lightbox: Use a friendlier format for the date

### LINKED ISSUE #45: lightbox: Use a friendlier format for the date, like "Yesterday at 4:47 PM"
…or "A few seconds ago", etc.

Currently it shows something like "Mar 31, 2023 15:09:51".

### PR DESCRIPTION
**Pull request by mofirojean**
This PR resolves the issue of displaying a user friendly date format in the lightbox appbar rather than showing the normal timestamp.

fixes #45

### CODE REVIEW

File: lib/widgets/lightbox.dart [hunk 1]
```diff
@@ -142,6 +142,28 @@ class _LightboxPageLayoutState extends State<_LightboxPageLayout> {
...
+      final time = DateFormat.jm(locale).format(messageDate);
+      return "Today at $time";
+    } else if (difference.inHours < 48) {
+      final time = DateFormat.jm(locale).format(messageDate);
+      return "Yesterday at $time";
+    } else {
+      final date = DateFormat('MMM d, yyyy', locale).format(messageDate);
+      final time = DateFormat('hh:mm a', locale).format(messageDate);
+      return "$date at $time";
+    }
```
Comment (chrisbobbe) [inline lib/widgets/lightbox.dart:164]: These are untranslated English strings. Instead, we should show strings in the user's chosen language.

Comment (mofirojean) [reply lib/widgets/lightbox.dart:164]: Thank you for the feedback, @chrisbobbe. 
Please can you provide a suggestion on how i should achieve that?

Comment (chrisbobbe) [reply lib/widgets/lightbox.dart:164]: I'm not able to prioritize that right now; this is a post-launch issue. Maybe there's a library out there that can handle this for us?

Comment (mofirojean) [reply lib/widgets/lightbox.dart:164]: Okay thank you, I will look that up.

**Comment by gnprice** [comment]
Closing this to get it out of our queue. @mofirojean if you find time in the future to revise this PR to address the feedback above (and if the issue is still open at that point), please feel free to re-open.

**Pull request closed without merging**